src/Mod/PartDesign/App/FeatureExtrude.cpp

```diff
@@ -51,18 +51,20 @@
 #include <Base/Exception.h>
 #include <Base/Placement.h>
 #include <Base/Reader.h>
+#include <Base/Tools.h>
 
 #include "FeatureExtrude.h"
 
 using namespace PartDesign;
 
-
 PROPERTY_SOURCE(PartDesign::FeatureExtrude, PartDesign::ProfileBased)
 
 App::PropertyQuantityConstraint::Constraints FeatureExtrude::signedLengthConstraint = { -DBL_MAX, DBL_MAX, 1.0 };
+const double maxAngle = 90 - Base::toDegrees<double>(Precision::Angular());
```

Comment:
Avoid global variables. You can declare it as a static variable and move it to the constructor of FeatureExtrude.

Reply:
Thanks. I made it now a stat class member.